lib/vector-core/buffers/examples/buffer_perf.rs

```diff
@@ -1,9 +1,9 @@
 #![allow(clippy::print_stderr)] // soak framework
 #![allow(clippy::print_stdout)]
 // Clippy allows are because this is an example/soak-y test, where we don't
-// actually care about the presence of `println!` calls.
+// actually care about the presence of `info!` calls.
```

Comment:
Regex replace?

Reply:
Good catch.

Reply:
We don't even use `println!` calls in `buffer_perf` anymore, so I can actually just remove those allowances entirely... :D